assets/package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "deploy": "webpack --mode production",
     "stylelint": "stylelint 'css/**/*.scss'",
-    "watch": "webpack --mode development --watch-stdin"
+    "watch": "webpack --mode development --watch-stdin",
+    "lint": "./node_modules/.bin/eslint ."
```

Comment:
Is `./node_modules/.bin` to run the project-local executable? If so, I think we can swap that for [`npx`](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b)? I usually run stylelint via npx, I must’ve forgotten when I added the `stylelint` script on Line 6.